docs/METRICS.md

```diff
@@ -49,6 +49,12 @@ Use this metric to determine how long messages are taking to be processed.
 
 Golden metric type: latency.
 
+### message_retry_count
+
+Use this metric to determine how many retries performed for message processing.
+
+Golden metric type: QoS.
```

Comment:
I think this page should say "Golden signal" not "Golden metric": 

https://blog.netsil.com/the-4-golden-signals-of-api-health-and-performance-in-cloud-native-applications-a6e87526e74

I think there are 4 types of golden signal, I don`t think QoS is one.